docs/cql/functions.rst

```diff
@@ -261,6 +261,10 @@ It also can be used to count the non-null value of a given column::
 
     SELECT COUNT (scores) FROM plays;
 
+.. note::
+    Counting all rows in a table may be time-consuming and exceed the default timeout. In such a case, 
+    see :doc:`Counting all rows in a table is slow </kb/count-all-rows>` for instructions.
+
```

Comment:
This is missing on query autopararallization (fe65122ccd40a2a3577121aebdb9a5b50deb4a90). It's still slow, but nowhere near as slow as before. @havaker

Reply:
The third patch in this series includes the text:
   ScyllaDB 5.1 includes improvements to speed up the execution of SELECT COUNT(*) queries. 
    To increase the count speed, we recommend upgrading to ScyllaDB 5.1 or later. 

But I think it is in the linked document, but not here.

Reply:
@avikivity @nyh It is my understanding that parallel execution of sub-queries is an improvement that will be available in version 5.1. If so, it is better to recommend upgrading ScyllaDB to 5.1 rather than explain how the improvement can be used in earlier versions - UNLESS there are tons of users who will prefer to speed up the COUNT queries in earlier versions. If that's the case, let me know, and I'll document it (preferably in another PR so that we can move on with this one).

Reply:
This is documenting master (5.2), not 5.1 and not earlier versions. The 5.0 documentation will not reflect the result of this patch.